__tests__/app.test.js

```diff
@@ -46,3 +46,45 @@ describe("/api", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id", () => {
+  test("GET 200 responds article by article_id", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
```

Comment:
Using more flexible jest matchers such as toMatchObject or objectContaining keep the tests flexible if the test data was to change or extend in properties in the future. The way you have approached this is still valid, as it won't fail if other properties are added in the future but thought I'd let you know. Here is a link if you're interested: https://jestjs.io/docs/expect#tomatchobjectobject